modules/promotions/admin-menu-items/custom-code-promotion-item.php

```diff
@@ -6,7 +6,7 @@
 	exit; // Exit if accessed directly
```

Comment:
@elchugreeva I've moved the base for this test into PRO on [ED-13558]

Can you use `use ElementorPro\Modules\Tiers\AdminMenuItems\Base_Promotion_Template;` once this merges?

Reply:
@Barcov, this is a core task.
Can you please elaborate?

Reply:
@davseve I see now it's Free to Advanced. So this is not needed.